src/styles/style.ts

```diff
@@ -47,8 +32,8 @@ export const GlobalStyle = (theme: Theme): SerializedStyles => css`
   }
 
   .MuiFormControl-root {
```

Comment:
im wondering if all these styles below here (lines 34 to 93) should also be moved out of the global styles into individual components. perhaps not if they are relevant for many components.